src/api/app/controllers/staging/workflows_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Staging::WorkflowsController < ApplicationController
+  before_action :require_login
+  before_action :set_project
+
+  def create
+    staging_workflow = @project.build_staging
+    authorize staging_workflow
+
+    staging_workflow.managers_group = Group.find_by!(title: xml_hash['managers'])
```

Comment:
What happens if this goes wrong?

Reply:
https://github.com/openSUSE/open-build-service/pull/7161/files#diff-dd30d7dc99a89d91d1f36eeda51d9730R49

Reply:
It ends in the application controller and returns a 404 with "Couldn't find Group"